target/linux/ipq40xx/files-5.4/arch/arm/boot/dts/qcom-ipq4019-wtr-m2133hp.dts

```diff
@@ -0,0 +1,413 @@
+			switch_lan_bmp = <0x1c>;
+		};
+
+		ess-psgmii@98000 {
+			status = "okay";
+		};
+
+		edma@c080000 {
+			status = "okay";
+		};
+
+		usb2@60f8800 {
+			status = "okay";
+		};
+
+		usb3@8af8800 {
+			status = "okay";
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		power_white: power_white {
+			label = "wtr-m2133hp:white:power";
+			gpios = <&tlmm 40 GPIO_ACTIVE_HIGH>;
+		};
+
+		power_orange: power_orange {
+			label = "wtr-m2133hp:orange:power";
+			gpios = <&tlmm 25 GPIO_ACTIVE_HIGH>;
+		};
+
+		power_blue: power_blue {
+			label = "wtr-m2133hp:blue:power";
+			gpios = <&tlmm 43 GPIO_ACTIVE_HIGH>;
+		};
+
+		router_white {
+			label = "wtr-m2133hp:white:router";
+			gpios = <&tlmm 28 GPIO_ACTIVE_HIGH>;
+		};
+
+		router_orange {
+			label = "wtr-m2133hp:orange:router";
+			gpios = <&tlmm 46 GPIO_ACTIVE_HIGH>;
+		};
+
+		internet_white {
+			label = "wtr-m2133hp:white:internet";
+			gpios = <&tlmm 27 GPIO_ACTIVE_HIGH>;
+		};
+
+		internet_orange {
+			label = "wtr-m2133hp:orange:internet";
+			gpios = <&tlmm 45 GPIO_ACTIVE_HIGH>;
+		};
+
+		wireless_white {
+			label = "wtr-m2133hp:white:wireless";
+			gpios = <&tlmm 24 GPIO_ACTIVE_HIGH>;
+		};
+
+		wireless_orange {
+			label = "wtr-m2133hp:orange:wireless";
+			gpios = <&tlmm 44 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		auto_mode {
+			label = "auto_mode";
+			gpios = <&tlmm 9 GPIO_ACTIVE_LOW>;
+			linux,code = <BTN_0>;
+			linux,input-type = <EV_SW>;
+		};
+
+		router_mode {
+			label = "router_mode";
+			gpios = <&tlmm 10 GPIO_ACTIVE_LOW>;
+			linux,code = <BTN_1>;
+			linux,input-type = <EV_SW>;
+		};
+
+		ap_mode {
+			label = "ap_mode";
+			gpios = <&tlmm 11 GPIO_ACTIVE_LOW>;
+			linux,code = <BTN_2>;
+			linux,input-type = <EV_SW>;
+		};
+
+		reset {
+			label = "reset";
+			gpios = <&tlmm 18 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		wps {
+			label = "AOSS Button";
+			gpios = <&tlmm 32 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+	};
+};
+
+&tlmm {
+	serial_0_pins: serial0_pinmux {
+		mux {
+			pins = "gpio16", "gpio17";
+			function = "blsp_uart0";
+			bias-disable;
+		};
+	};
+
+	mdio_pins: mdio_pinmux {
+		mux_1 {
+			pins = "gpio6";
+			function = "mdio";
+			bias-pull-up;
+		};
+
+		mux_2 {
+			pins = "gpio7";
+			function = "mdc";
+			bias-pull-up;
+		};
+	};
+
+	nand_pins: nand_pins {
+		pullups {
+			pins = "gpio52", "gpio53", "gpio58",
+				"gpio59";
+			function = "qpic";
+			bias-pull-up;
+		};
+
+		pulldowns {
+			pins = "gpio54", "gpio55", "gpio56",
+				"gpio57", "gpio60", "gpio61",
+				"gpio62", "gpio63", "gpio64",
+				"gpio65", "gpio66", "gpio67",
+				"gpio68", "gpio69";
+			function = "qpic";
+			bias-pull-down;
+		};
+	};
+};
+
+&blsp_dma {
+	status = "okay";
+};
+
+&blsp1_uart1 {
+	pinctrl-0 = <&serial_0_pins>;
+	pinctrl-names = "default";
+	status = "okay";
+};
+
+&cryptobam {
+	status = "okay";
+};
+
+&pcie0 {
+	status = "okay";
+
+	bridge@0,0 {
+		reg = <0 0 0 0 0>;
```

Comment:
Please use `<0x00000000 0 0 0 0>` like other devices.
And I haven't checked it yet, but you'll probably need `address-cells` and `size-cells` here to suppress warning during compliation of the dts. (Please check `logs/target/linux/install.txt`.)

Reply:
Adding `address-cells` and `size-cells` fixes compilation warning. Thanks.

Reply:
BTW it would have been fair to mention this compilation warning in the first place, instead of hiding it. Are there other compilation warning we should know of?

Reply:
I checked build logs and found no error or warning related to this PR.